esp8266-fastled-webserver/Fields.cpp

```diff
@@ -15,10 +15,21 @@
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
+#include "common.h"
```

Comment:
SIMPLE FILE REVIEW

But, this file **_DOES_** have more than one comment.

Adds the private strings that only this file needs (originally found in `field.h`) directly to this file.